docs/sources/write/image-guidelines/index.md

```diff
@@ -135,7 +135,11 @@ However, ensure that the screenshot accurately reflects the directions and value
 
 Consult the following guidelines when you create screenshots:
 
-- **Size**: The maximum width of a screenshot is 600 pixels.
+- **Size**: The maximum width of a screenshot is 750 pixels.
```

Comment:
```suggestion
- **Size**: The maximum width of a screenshot is 750 pixels. Ideally, screenshots are 2:1 density, or 1500px.
```

We probably want to inform writers to use a higher density image than the content container.